docs/src/main/asciidoc/reactive-routes.adoc

```diff
@@ -667,6 +667,31 @@ public void init(@Observes io.vertx.mutiny.ext.web.Router router) {
 }
 ----
 
+=== Well known route order values
```

Comment:
Hm, it's a `vertx-http` thing but it's documented in the reactive routes reference guide. I wonder if we should place this information in the [HTTP Reference ](https://quarkus.io/guides/http-reference) and add a link here instead?

Reply:
Yes, wrong place, it should be in the HTTP reference.

Reply:
Moved to the HTTP reference, added a new chapter at the end.